src/ansible/roles/cleanup/tasks/main.yml

```diff
@@ -29,4 +29,4 @@
 
   - name: Remove SSSD's database and logs
     shell: rm -f /var/lib/sss/db/* /var/lib/sss/mc/* /var/log/sssd/*
-  when: inventory_hostname == 'client.test' or inventory_hostname == 'master.ipa.test'
+  when: inventory_hostname == 'client.test' or inventory_hostname == 'master.ipa.test' or inventory_hostname == 'master2.ipa2.test' or inventory_hostname == 'master2.ipa2.test'
```

Comment:
This should run on group ipa and client.

Reply:
Fixed.